src/krb5_asrep_fmt_plug.c

```diff
@@ -367,7 +553,7 @@ struct fmt_main fmt_krb5asrep = {
 		SALT_ALIGN,
 		MIN_KEYS_PER_CRYPT,
 		MAX_KEYS_PER_CRYPT,
-		FMT_CASE | FMT_8_BIT | FMT_UNICODE | FMT_UTF8 | FMT_OMP | FMT_DYNA_SALT,
+		FMT_CASE | FMT_8_BIT | FMT_UNICODE | FMT_UTF8 | FMT_OMP,
 		{NULL},
```

Comment:
Given your "large enough" of 4096 (even when changed back to dyna_salt), I presume FMT_HUGE_INPUT is required as well.

Reply:
Yes, this particular change is not good.

However, I was running into the "Could not find salt in db - salt() return inconsistent?" error message with dyna_salt stuff. 

I can post a dyna_salt version of this PR to get some debugging help.

Reply:
Please do. You shouldn't break things just to work around your own bugs 😉